glide.lock

```diff
@@ -1,215 +1,16 @@
-hash: 9589cf38139f18ca918955308856f05f0f5196ca6189cd6961ffd1aff19d8ba6
```

Comment:
I'm just trying to upgrade openshift + k8s dependencies right now. Its a bit fiddly - I'm about to push a `Makefile` change that updates the latest glide.yaml file changes